vllm_detector_adapter/generative_detectors/base.py

```diff
@@ -217,9 +217,9 @@ def calculate_scores(self, response: ChatCompletionResponse) -> List[float]:
                 # per discussion with granite team. So we are pulling in `top_logprobs`
                 for top_logprob in logprob_info_i.top_logprobs:
                     token = top_logprob.token
-                    if token.strip().lower() == self.SAFE_TOKEN.lower():
```

Comment:
any particular reason to remove this ?  I see that we have also changed the case in the `safe_token` definition, but since its in base class, it would be safer to compare `.lower()` with `.lower()`, otherwise, we might end up checking not lower case with lower case